tailwind.config.js

```diff
@@ -70,7 +70,7 @@ module.exports = {
         '40': '10rem'
       },
       screens: {
-        xxl: '2048px'
+        xxl: '1536px'
```

Comment:
This doesn't have any consequences across the site?
Why do we need this update for the modal?

Reply:
- It does not have consequences at all, we could consider 1536px as a big screen. `AnnouncementModal` is the only component in the whole app using `xxl:`
- I was aiming to use the same breakpoints as tailwind https://tailwindcss.com/docs/breakpoints